lib/Varien/Data/Form/Element/Fieldset.php

```diff
@@ -41,6 +43,11 @@ class Varien_Data_Form_Element_Fieldset extends Varien_Data_Form_Element_Abstrac
      */
     protected $_sortChildrenByKey = '';
 
+    /**
+     * @var string
+     */
+    protected $_sortDirection = '';
```

Comment:
This is a logic change, which prevents the current behaviour of this value beeing part of _data over __set and __get, which also means you change the visibility from public to protected.

Reply:
Is it intentional logic or just a missed field? There arn't getters/setters for this. I'll revert.

Reply:
It may not be intentional, but its there for a long time, so there may be a lot of people who did make use of it in the way it is currently over the getData()